test/pbbuilder_test.rb

```diff
@@ -25,11 +25,38 @@ class PbbuilderTest < ActiveSupport::TestCase
 
     assert_equal "Hello world", person.name
     assert_equal "Friend #1", person.friends.first.name
-    assert_equal ["ok", "that's", "cool"], person.field_mask.paths
+    assert_equal ["cool"], person.field_mask.paths
     assert_equal "Manuelo", person.best_friend.name
     assert_equal "Eggs", person.favourite_foods["Breakfast"]
   end
 
+  test "duplicated enum fields don't happen" do
+    person = Pbbuilder.new(API::Person.new) do |pb|
+      pb.field_mask do
+        pb.paths ["ok", "that's"]
+      end
+    end.target!
+
+    p = Pbbuilder.new(person) do |pb|
+      pb.field_mask do
+        pb.paths ["ok", "that's"]
+      end
+    end.target!
+
+    assert_equal(["ok", "that's"], p.field_mask.paths)
+  end
+  
+  test "Would not stack duplicated values, uses last one defined" do
```

Comment:
nit: "would not" is not quite what it is - "replaces the repeated field's value with the last one set"